src/main/java/dinkplugin/notifiers/PetNotifier.java

```diff
@@ -22,18 +37,52 @@ protected String getWebhookUrl() {
     }
 
     public void onChatMessage(String chatMessage) {
-        if (isEnabled() && PET_REGEX.matcher(chatMessage).matches()) {
-            this.handleNotify();
+        if (petName == null && isEnabled() && PET_REGEX.matcher(chatMessage).matches()) {
+            this.petName = "";
```

Comment:
Small readability improvement, makes it clear what setting the pet name actually does. Wording up for improvement

```suggestion
            // Prime the notifier to trigger next tick
            this.petName = "";
```